### PR TITLE
[3.2.0] Add simplified setup for WSO2 API Manager

### LINKED ISSUE #428: [3.2.0] Add Simplified setup for WSO2 API Manager
**Description:**
It is required to add the Simplified setup for WSO2 API Manager 3.2.0.

**Affected Product Version:**
APIM 3.2.0

### PR DESCRIPTION
**Pull request by ThilinaManamgoda**
Tested with, 
>- Analytics Dashboard `3.2.0-RC1`
>- All the other component `3.2.0-beta`

### CODE REVIEW

**Comment by chirangaalwis** [comment]
@ThilinaManamgoda the PR LGTM!

As per the product team, the `RC-2` release is expected to happen soon. Let's finalize everything using the same release (both APIM and APIM Analytics) and merge this.

**Comment by ThilinaManamgoda** [comment]
> @ThilinaManamgoda the PR LGTM!
> 
> As per the product team, the `RC-2` release is expected to happen soon. Let's finalize everything using the same release (both APIM and APIM Analytics) and merge this.

ACK

**Comment by ThilinaManamgoda** [comment]
> > @ThilinaManamgoda the PR LGTM!
> > As per the product team, the `RC-2` release is expected to happen soon. Let's finalize everything using the same release (both APIM and APIM Analytics) and merge this.
> 
> ACK

Teste with RC6 distributions. This PR also address fixes #329

File: simple/README.md [hunk 1]
```diff
@@ -11,15 +11,17 @@
...
+ This artifact doesn't support [Docker Desktop](https://www.docker.com/products/docker-desktop). 
 
-* WSO2 subscribed users can run [wso2am-latest.sh](https://github.com/wso2/kubernetes-apim/blob/3.0.x/simple/deployment-scripts/wso2am-latest.sh) with the latest updates by providing their subscription Username and Password. If you do not possess an active WSO2 subscription already, run [wso2am-ga.sh](https://github.com/wso2/kubernetes-apim/blob/3.0.x/simple/deployment-scripts/wso2am-ga.sh) which does not require subscription credentials. Further users can acquire a subsciption by refering this [link](https://wso2.com/subscription).
+* WSO2 subscribed users can run [wso2am-latest.sh](https://github.com/wso2/kubernetes-apim/blob/3.2.x/simple/deployment-scripts/wso2am-latest.sh) with the latest updates by providing their subscription Username and Password. If you do not possess an active WSO2 subscription already, run [wso2am-ga.sh](https://github.com/wso2/kubernetes-apim/blob/3.2.x/simple/deployment-scripts/wso2am-ga.sh) which does not require subscription credentials. Further users can acquire a subsciption by refering this [link](https://wso2.com/subscription).
```
Comment (chirangaalwis) [inline simple/README.md:17]: @ThilinaManamgoda the word `referring` is not spelled correctly, here.

File: simple/README.md [hunk 2]
```diff
@@ -11,15 +11,17 @@
 
 * Install [Kubernetes  Client](https://kubernetes.io/docs/tasks/tools/install-kubectl/) in order to run the steps provided in the following Quick Start Guide.
 
-* An already setup Kubernetes cluster. If you haven’t already setup a K8s cluster  Refer [the guide](https://kubernetes.io/docs/setup/).
+* An already setup Kubernetes cluster. If you haven’t already setup a K8s cluster  Refer [the guide](https://kubernetes.io/docs/setup/learning-environment/minikube/).
+ This artifact doesn't support [Docker Desktop](https://www.docker.com/products/docker-desktop).
```
Comment (chirangaalwis) [inline simple/README.md:15]: @ThilinaManamgoda shall we add this as a note, as follows?

> This artifact doesn't support [Docker Desktop](https://www.docker.com/products/docker-desktop).

**Review by chirangaalwis — CHANGES_REQUESTED** [review]

File: simple/README.md (hunk 1 above)
Comment (chirangaalwis) [reply simple/README.md:17]: @ThilinaManamgoda also it has to be `Further users can acquire a subscription by referring to this [link](https://wso2.com/subscription).`

**Review by chirangaalwis — APPROVED** [review]

**Pull request merged**